jekyll/_cci2/ecs-ecr.md

```diff
@@ -0,0 +1,522 @@
+
+## Prerequisites
+
+### Use Terraform to Create AWS Resources
+
+Several AWS resources are required
+to build and deploy the application in this guide.
+CircleCI provides [several Terraform scripts](https://github.com/CircleCI-Public/circleci-demo-aws-ecs-ecr/tree/master/terraform_setup)
+to create these resources.
+To use these scripts,
+follow the steps below.
+
+1. [Create an AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
+2. [Install Terraform](https://www.terraform.io/).
+3. Clone the [sample project](https://github.com/CircleCI-Public/circleci-demo-aws-ecs-ecr)
+and go to its root directory.
+4. Update `~/terraform_setup/terraform.tfvars` with real values for the AWS variables.
+For more details,
+see the [Configure CircleCI Environment Variables](#configure-circleci-environment-variables) section below.
+5. Create the AWS resources
+by running the following commands.
+
+```bash
+cd terraform_setup
+terraform init
+terraform plan  # review the plan
+terraform apply  # apply the plan and create AWS resources
+```
+
+**Note:**
+You can destroy most AWS resources
+by running `terraform destroy`.
+If any resources remain,
+check the [AWS Management Console](https://console.aws.amazon.com/),
+particularly the **ECS**, **CloudFormation** and **VPC** pages.
+
+### Configure CircleCI Environment Variables
+
+In the CircleCI application,
+set the following [project environment variables]({{ site.baseurl }}/2.0/env-vars/#setting-an-environment-variable-in-a-project).
+
+Variable                 | Description
+-------------------------|------------
+AWS_ACCESS_KEY_ID        | Security credentials for AWS.
+AWS_SECRET_ACCESS_KEY    | Security credentials for AWS.
+AWS_DEFAULT_REGION       | Used by the AWS CLI.
+AWS_ACCOUNT_ID           | Required for deployment. [Find your AWS Account ID](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html#FindingYourAWSId).
+AWS_RESOURCE_NAME_PREFIX | Prefix for some required AWS resources. Should correspond to the value of `aws_resource_prefix` in `terraform_setup/terraform.tfvars`.
+{:class="table table-striped"}
+
+## Configuration Walkthrough
+
+Every CircleCI project requires a configuration file called [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/).
+Follow the steps below to create a complete `config.yml` file.
+
+### Specify a Version
+
+Every `config.yml` starts with the [`version`]({{ site.baseurl }}/2.0/configuration-reference/#version) key.
+This key is used
+to issue warnings about breaking changes.
+
+```yaml
+version: 2
+```
+
+### Create a Build Job
+
+A run is comprised of one or more [jobs]({{ site.baseurl }}/2.0/configuration-reference/#jobs).
+For this sample application,
+there are two jobs:
+one for building and testing the application,
+and one for deploying the application.
+These jobs are called `build` and `job`, respectively.
```

Comment:
Should it be `deploy` rather than `job`?

Reply:
Yes it should be! Great catch :)